package.json

```diff
@@ -9,7 +9,8 @@
     "build": "npm run tslint && tsc",
     "tslint": "tslint --project tsconfig.json -r tslint.json --fix || true",
     "test": "npm run clean && karma start --singleRun",
-    "test:watch": "npm run clean && karma start"
+    "test:watch": "npm run clean && karma start",
+    "test:browser": "npm run clean && karma start --browsers=Chrome --single-run=false --auto-watch"
```

Comment:
This is a breaking change. Version should bump; perhaps a minor bump to `0.2.0` since major is `0`. Should also document breaking changes. Perhaps in the README, or in some other document.

Reply:
hows is that a breaking change? I'm just adding a new command.

Reply:
The entire PR is a breaking change, not just this line.

Reply:
updated the version. 
The parts I'm changing were not a part of README before, should I add a section about blocking? Or just add a changelog file?